core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -173,8 +173,6 @@ protected[controller] class RestAPIVersion(apipath: String, apiversion: String)(
                     webexp.routes()
                 } ~ {
                     swaggerRoutes
-                } ~ options {
```

Comment:
Could add it back, but experimental features don't guaranteed that API breaking changes won't happen. @csantanapr, what do you think about removing the OPTIONS verb from experimental?